### DISCUSSION TITLE
TypeError: webpack.optimize.CommonsChunkPlugin is not a constructor

### DISCUSSION THREAD

**Discussion by knightswatch3** [webpack/webpack#13025] [Q&A]
I receive this error when I am using commonsChunkPlugin in the plugins array. I was assuming this to be not part of plugins in webpack 5 but the documentation says otherwise. 

What is the new way of conveying about chunks to webpack in v5 ?

**Comment by alexander-akait** [comment]
`webpack.optimize.CommonsChunkPlugin` is deprecated and removed, please use https://webpack.js.org/plugins/split-chunks-plugin/
